Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+      ]
+    },
+    "ObjEventLaserCancelCollisionSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionLength", "type": "float32" }
+      ]
+    },
+    "ObjPrismSpawner": {
+      "fields": [
+        { "name": "Pitch", "type": "float32" },
+        { "name": "Yaw", "type": "float32" },
+        { "name": "Target", "type": "vector3" },
+        { "name": "ObjectID", "type": "object_reference" },
+        { "name": "Type", "type": "ObjPrismSpawner::Type" },
+        { "name": "CameraID", "type": "object_reference" }
+      ]
+    },
+    "ObjFiberSpawner": {
+      "fields": [
+        { "name": "CollisionWidth", "type": "float32" },
+        { "name": "CollisionHeight", "type": "float32" },
+        { "name": "CollisionDepth", "type": "float32" },
+        { "name": "PathID", "type": "uint32" },
+        { "name": "TargetCamera", "type": "object_reference" },
+        { "name": "EventType", "type": "ObjFiberSpawner::EventType" },
+        { "name": "IsExitOutside", "type": "bool" },
+        { "name": "IsExit2D", "type": "bool" },
+        { "name": "ExitType", "type": "ObjFiberSpawner::ExitType" },
+        { "name": "LaserSpeed", "type": "float32" },
+        { "name": "IsChangeSpeed", "type": "bool" }
+      ]
+    },
+    "ObjCogWheelSpawner": {
+      "fields": [
+        { "name": "num", "type": "uint32" }
+      ]
+    },
+    "ObjRotationGlobeSpawner": {
+      "fields": [
+        { "name": "SizeType", "type": "ObjRotationGlobeSpawner::SizeType" },
+        { "name": "IsReverseStart", "type": "bool" }
+      ]
+    },
+    "ObjPuzzleRingSpawner": {
+      "fields": [
+        { "name": "Type", "type": "ObjPuzzleRingSpawner::Type" }
+      ]
+    },
+    "ObjSpinLaserBaseSpawner": {
+      "fields": [
+        { "name": "AngleVelocity", "type": "float32" },
+        { "name": "LaserLength", "type": "float32" },
+        { "name": "LaserNum", "type": "ObjSpinLaserBaseSpawner::LaserNum" },
+        { "name": "RayCast", "type": "bool" }
+      ]
+    },
+    "ObjSpinLaserSpawner": {
+      "parent": "ObjSpinLaserBaseSpawner",
+      "fields": [
+        { "name": "IsMessageTurn", "type": "bool" },
+        { "name": "TurnType", "type": "ObjSpinLaserSpawner::TurnType" },
+        { "name": "ThresholdAngle", "type": "float32" }
+      ]
+    },
+    "ObjSpinLaserPathMoveSpawner": {
+      "parent": "ObjSpinLaserBaseSpawner",
+      "fields": [
+        { "name": "TurnType", "type": "ObjSpinLaserPathMoveSpawner::TurnType" },
+        { "name": "ThresholdAngle", "type": "float32" },
+        { "name": "IsRepeat", "type": "bool" },
+        { "name": "IsMessageMove", "type": "bool" },
+        { "name": "IsMessageTurn", "type": "bool" },
+        { "name": "PathID", "type": "uint32" },
+        { "name": "MoveDistance", "type": "float32" },
+        { "name": "MoveSpeed", "type": "float32" }
+      ]
+    },
```

Comment:
Unused object. Parameters named based on in-game functionality and the version used in the retail game `SpinLaser`.